.dockerignore

```diff
@@ -0,0 +1,7 @@
+Dockerfile
+.gitignore
+.travis.yml
+*~
```

Comment:
Emacs backup files.

Hmm, since we're using a newer Docker, maybe they have added support for "ignore anywhere" and we don't have to repeat this for all levels of the directory hierarchy.